src/_guides/language/language-tour.md

```diff
@@ -282,19 +289,28 @@ Any variable can have a constant value.
 
 <?code-excerpt "misc/lib/language_tour/variables.dart (const-vs-final)"?>
 {% prettify dart %}
-// Note: [] creates an empty list.
-// const [] creates an empty, immutable list (EIL).
-var foo = const []; // foo is currently an EIL.
-final bar = const []; // bar will always be an EIL.
-const baz = []; // baz is a compile-time constant EIL.
+var foo = const [];
+final bar = const [];
+const baz = []; // Equivalent to `const []`
+{% endprettify %}
+
+You can omit `const` from the initializing expression of a `const` declaration,
+like for `baz` above. For details, see [DON’T use const redundantly][].
 
-// You can change the value of a non-final, non-const variable,
-// even if it used to have a const value.
-foo = [];
+You can change the value of a non-final, non-const variable,
+even if it used to have a const value:
 
-// You can't change the value of a final or const variable.
-// bar = []; // Unhandled exception.
-// baz = []; // Unhandled exception.
+<?code-excerpt "misc/lib/language_tour/variables.dart (reassign-to-non-final)"?>
+{% prettify dart %}
+foo = [1, 2, 3]; // Was const []
+{% endprettify %}
+
+Being compile-time constants, you can't assign to a const variable:
+
+{:.fails-sa}
+<?code-excerpt "misc/lib/language_tour/variables.dart (cant-assign-to-const)"?>
+{% prettify dart %}
+baz = []; // Error: Constant variables can't be assigned a value.
```

Comment:
Can we avoid using "assigned" in this comment, or is that in the error message?

Reply:
That is the actual error message.

Reply:
Ah, darn it all.

Reply:
(FWIW, I think that "assigned" is the proper term here. As you pointed out earlier, the other occurrence of "=" is actually part of an initializing declaration.)